ec/src/hashing/tests.rs

```diff
@@ -6,9 +6,9 @@ fn test_parity_of_prime_field_elements() {
     let a1 = Fq::from(0);
     let a2 = Fq::from(1);
     let a3 = Fq::from(10);
-    assert_eq!(parity(&a1), false);
-    assert_eq!(parity(&a2), true);
-    assert_eq!(parity(&a3), false);
+    assert!(!parity(&a1));
+    assert!(parity(&a2));
+    assert!(!parity(&a3));
```

Comment:
I think for these cases it's a bit easy to miss the `!`, so I prefer the old way

Reply:
fixed